boards/arm/b_l072z_lrwan1/doc/b_l072z_lrwan1.rst

```diff
@@ -0,0 +1,225 @@
+     :height: 325px
+     :alt: B-L072Z-LRWAN1
+
+More information about the board can be found at the `B-L072Z-LRWAN1 website`_.
+
+Hardware
+********
+
+The STM32L072CZ SoC provides the following hardware IPs:
+
+- Ultra-low-power (down to 0.29 µA Standby mode and 93 uA/MHz run mode)
+- Core: ARM |reg| 32-bit Cortex |reg|-M0+ CPU, frequency up to 32 MHz
+- Clock Sources:
+
+        - 1 to 32 MHz crystal oscillator
+        - 32 kHz crystal oscillator for RTC (LSE)
+        - Internal 16 MHz factory-trimmed RC ( |plusminus| 1%)
+        - Internal low-power 37 kHz RC ( |plusminus| 5%)
+        - Internal multispeed low-power 65 kHz to 4.2 MHz RC
+- RTC with HW calendar, alarms and calibration
+- Up to 24 capacitive sensing channels: support touchkey, linear and rotary touch sensors
+- 11x timers:
+
+        - 2x 16-bit with up to 4 channels
+        - 2x 16-bit with up to 2 channels
+        - 1x 16-bit ultra-low-power timer
+        - 1x SysTick
+        - 1x RTC
+        - 2x 16-bit basic for DAC
+        - 2x watchdogs (independent/window)
+- Up to 84 fast I/Os, most 5 V-tolerant.
+- Memories
+
+        - Up to 192 KB Flash, 2 banks read-while-write, proprietary code readout protection
+        - Up to 20 KB of SRAM
+        - External memory interface for static memories supporting SRAM, PSRAM, NOR and NAND memories
+- Rich analog peripherals (independent supply)
+
+        - 1x 12-bit ADC 1.14 MSPS
+        - 2x 12-bit DAC
+        - 2x ultra-low-power comparators
+- 11x communication interfaces
+
+        - USB OTG 2.0 full-speed, LPM and BCD
+        - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+        - 4x USARTs (ISO 7816, LIN, IrDA, modem)
+        - 6x SPIs (4x SPIs with the Quad SPI)
+- 7-channel DMA controller
+- True random number generator
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
+
+More information about STM32L072CZ can be found here:
+
+        - `STM32L072CZ on www.st.com`_
+        - `STM32L0x2 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr B-L072Z-LRWAN1 Discovery board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
```

Comment:
This renders better as:

    The default configuration can be found in the defconfig file:
    ``boards/arm/b_l072z_lrwan1/b_l072z_lrwan1_defconfig``